### DISCUSSION TITLE
Fix deprecation log messages

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#45405] [Q&A]
There are many deprecation log messages in my Symfony project. How can I fix them?

> The "Symfony\Bridge\Doctrine\Logger\DbalLogger" class implements "Doctrine\DBAL\Logging\SQLLogger" that is deprecated Use {@see \Doctrine\DBAL\Logging\Middleware} or implement {@see \Doctrine\DBAL\Driver\Middleware} instead.

> User Deprecated: The Liip\ImagineBundle\Templating\FilterTrait trait is deprecated since version 2.7 and will be removed in 3.0; use Twig instead.

> User Deprecated: The Liip\ImagineBundle\Templating\FilterExtension class is deprecated since version 2.7 and will be removed in 3.0; configure "liip_imagine.twig.mode" to "lazy" instead.

**Comment by ghost** [comment]
You don't "fix" a deprecation log. This is an informational log. You either upgrade your code base or turn off deprecation logs in configuration.
